### ISSUE TITLE
Files are downloaded as a folder with the files name

### ISSUE THREAD

**Issue by warriotox** [icetee/remote-ftp#1026]
### Prerequisites

* [ X] Put an X between the brackets on this line if you have done all of the following:
    * Checked that your issue isn't already filed: https://github.com/icetee/remote-ftp/issues

### Description
I'm having issues with all files. They seem to be processed as folders when trying to access them directly but when downloading the parent folder they are processed correctly as files. 

This is the same as https://github.com/icetee/remote-ftp/issues/1022 the issue still persists in 1.2.6.

Screenshot of create files: https://puu.sh/y0dp0/488af110f2.png
In 1.2.6 - https://puu.sh/y3ntw/cfb0dfaddd.png

For example when I try to download a /folder/style.css the progress bar is trying to download /folder/style.css/style.css

### Steps to Reproduce
Create a file on your FTP server and try downloading it with the remote-ftp plugin in Atom.

**Expected behavior:**
1. Downloading a file that doesnt exist on my computer from the server creates the file with the same name and contents.
2. Downloading the parent directory of my files creates/updates the files and the directory
3. Uploading the file to server updates the file on the server

**Actual behavior:** 
1. Downloading a file that doesnt exist on my computer from the server creates a folder instead.
2. Downloading the parent directory of my files creates files correctly
3. Uploading the file to server gets stuck.

**Reproduces how often:** [100%]

### Versions
remote-ftp - 1.2.6
Atom - 1.21.1
OS - Windows 10 Pro 64bit

### Additional Information

I'm not experiencing this behavior in IntelliJ Remote Host plugin, or FileZilla app..

**Comment by icetee** [comment]
Did the previously downloaded file have been removed?
What type of server do you have? vsftpd?

**Comment by warriotox** [comment]
Yes I did delete the file and try to download again with the same issue, even for files I didn't previously download. It only works correct when I download through the parent folder.
Our server is used for a Java library that our CMS uses. It's org.apache.ftpserver - ftpserver-core 1.0.6 https://mvnrepository.com/artifact/org.apache.ftpserver/ftpserver-core/1.0.6

**Comment by zoranvojinov** [comment]
Hi,
I have a similar Problem:
I'm connected to the ftp Server (sftp) but when trying to sync/download it just Downloads the folders not the files. Only whenn download the direct folder then the subfiles will be downloaded.

Is this corrected? Remote-ftp does not work otherwise.

Atom: 1.21.1
Remote-FTP: 1.2.6
SFTP
OS: Windows 7

**Comment by NamedDavidChan** [comment]
I have a same problem when I use freeSSHd as SFTP Server on Windows Server 2012.
Using FileZilla is normal.
